src/leader.py

```diff
@@ -31,6 +32,8 @@ class Leader:
 
     STRAIGHT_ANGLE = 90
     """The angle that the hardware associates as straight."""
+    DEMO_TURN_TIME = 2
+    """The time the vehicle takes to turn."""
```

Comment:
It does not take the vehicle two seconds to turn. This is how long you turn in demos. This needs a better comment.